common/aws/s3/client.go

```diff
@@ -106,24 +105,13 @@ func NewClient(ctx context.Context, cfg commonaws.ClientConfig, logger logging.L
 	return ref, err
 }
 
-func PeekObjectSize(ctx context.Context, s3Client *s3.Client, bucket, key string) (int64, error) {
-	input := &s3.HeadObjectInput{
-		Bucket: aws.String(bucket),
-		Key:    aws.String(key),
-	}
-	result, err := s3Client.HeadObject(ctx, input)
-	if err != nil {
-		return 0, fmt.Errorf("failed to head object: %w", err)
-	}
-	return *result.ContentLength, nil
-}
-
 func (s *client) DownloadObject(ctx context.Context, bucket string, key string) ([]byte, error) {
-	size, err := PeekObjectSize(ctx, s.s3Client, bucket, key)
-	if err != nil {
-		return nil, err
+	objectSize := 128 * 1024
```

Comment:
How was this number chosen? Also should make it a const

Reply:
Just give it a moderate number so it won't waste a lot, nor need to realloc the memory too many times.